tests/scripts/update_metadata.sh

```diff
@@ -12,10 +12,10 @@ readonly RELEASE_DIR='dist'
 gzip -k "${RELEASE_DIR}/blockly_compressed.js"
 gzip -k "${RELEASE_DIR}/blocks_compressed.js"
 
-blockly_size=$(wc -c < "${RELEASE_DIR}/blockly_compressed.js")
-blocks_size=$(wc -c < "${RELEASE_DIR}/blocks_compressed.js")
-blockly_gz_size=$(wc -c < "${RELEASE_DIR}/blockly_compressed.js.gz")
-blocks_gz_size=$(wc -c < "${RELEASE_DIR}/blocks_compressed.js.gz")
+blockly_size=$(wc -c < "${RELEASE_DIR}/blockly_compressed.js" | xargs)
```

Comment:
Why does `| xargs` mean whitespace gets stripped? Could  you add a comment to the source code?

Reply:
Done.